cmd/compute-domain-kubelet-plugin/device_state.go

```diff
@@ -115,13 +115,13 @@ func NewDeviceState(ctx context.Context, config *Config) (*DeviceState, error) {
 	}
 
 	for _, c := range checkpoints {
-		if c == DriverPluginCheckpointFile {
+		if c == DriverPluginCheckpointFileBasename {
```

Comment:
Unrelated change; just a variable rename. Was looking for the checkpoint file path by following code. Renamed this because a hard-learned lesson for me is to use the term "file" only for abstract file objects, and never for basenames or absolute paths. In this case, we store a basename.